packages/pds/src/db/migrations/20230724T194042238Z-optional-handle.ts

```diff
@@ -0,0 +1,51 @@
+import { Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<unknown>, dialect: Dialect): Promise<void> {
+  if (dialect === 'pg') {
```

Comment:
The paranoid part of me wants to flip this condition so that it's more likely to go down the `'pg'` path by mistake than the `'sqlite'` path, since the latter is quite destructive.
```suggestion
  if (dialect !== 'sqlite') {
```